cfgov/data_research/tests/management/commands/test_conference_export.py

```diff
@@ -0,0 +1,158 @@
+from django.test import TestCase
+from model_mommy import mommy
+from wagtail.wagtailcore.models import Page
+
+from data_research.models import ConferenceRegistration
+from data_research.management.commands.conference_export import (
+    ConferenceExporter, get_registration_form_from_slug
```

Comment:
Thanks, fixed in f0c1d20. I'm not sure why tests continued to pass both locally and on Travis above. It might be because the test file import failed and that doesn't cause the test runner to fail; maybe it just skips that file.